assembly/index.ts

```diff
@@ -180,9 +179,6 @@ export function finish(out: ArrayBuffer): void {
     let tempPtr   = changetype<usize>(temp);
 
     store8(bufferPtr, left, 0x80);
-    // for (let i = left + 1, len = padLength - 8; i < len; i++) {
-    //   store8(bufferPtr, i, 0);
-    // }
```

Comment:
What is the intention here?

Reply:
Are you prefer leave this commented block?

Reply:
Oh wow... I totally saw this and thought you commented it out. Woops all good :)